docs/checks/invalidMiniRoundaboutCheck.md

```diff
@@ -0,0 +1,127 @@
+# Invalid Mini Roundabout Check 
```

Comment:
Re: Bentleys comment about my markdown spiel

The initial purpose of these markdown files was to give a high level step by step overview of why the integrity check existed, and how it worked; which is great for non technical members of the community. After multiple rounds of enhancements, we realized that the code examples in the markdown and the checks code could easily get out of sync. Because of this, we want to minimize the amount of duplicate code blocks; making the description and live examples the most important piece of this file. If there's a particular helper function that you want to call out, feel free to describe it, but please leave out the code snippet. You can link to the particular checks code at the end of the document.

Thanks :) 

cc: @sayas01

Reply:
Thanks for the background information! That makes a lot of sense. I've removed the in-depth code review section. Does it look better now, or do you think I should explain the code, just at a higher level and without code snippets?

Reply:
Looks great !